### ISSUE TITLE
Explore the math behind frost to see if we can avoid having to specify signers when creating party sigs

### ISSUE THREAD

**Issue by xoloki** [Trust-Machines/frost#10]

**Comment by a3slade** [comment]
tl;dr - No.

z = sum( d_i + (rho_i * e_i) + lambda_i * s_i * H(R, Y, msg)) 
    // d, e, and s are known only to party i
    // lambda is computed based on the set of signers
    // Y is the group public key

R = sum( (d_i * G) + (rho_i * e_i * G) )
    // this all publicly known because dG and eG are publicly committed

Failed Attempt 1:
Do not include lambda in the signature. Multiple all of each z_i by lambda_i afterwards.
This results in d_i + rho_i * e_i also being multiplied by lambda_i.
We could back this out if we knew these values. But we don't. We only know their public commitments.

Failed Attempt 2:
Do the above but adjust R by multiplying each publicly committed nonce by that party's lambda.
This is effectively adjusting the publicly committed nonces to match the lambda-scaled nonces.
Now zG = R' + Y * H(Y, R, msg)
However, R' is not the same as the R used in the hash.
If we passed around more information (eg., the sig could include both R and R'),
then we could get this assertion to hold. But that removes the security as intended.

Failed Attempt 3:
I already erased it from my window. It was an attempt to resolve the conflicting Rs. It did not work out.
